cozy/cost_model.py

```diff
@@ -49,19 +84,26 @@ def order_objects(x, y) -> Order:
 
 class CostModel(object):
 
-    def __init__(self, assumptions : Exp = T, examples=(), funcs=(), freebies : [Exp] = []):
+    def __init__(self,
+            assumptions     : Exp   = T,
+            examples                = (),
+            funcs                   = (),
+            freebies        : [Exp] = [],
+            ops             : [Op]  = []):
```

Comment:
We should add a docstring to this constructor that describes what `freebies` and `ops` do.

Reply:
In which case should also describe what assumptions, examples, and funcs do as well?

Reply:
Yep!

Reply:
Please take a look at the docstring I added. I wasn't sure how specific I needed to be for them so I just gave a very brief description.